### DISCUSSION TITLE
配置文件中出站代理的ip-version选项的含义

### DISCUSSION THREAD

**Discussion by smy116** [MetaCubeX/mihomo#1174] [Q&A] [answered]
Mihomo的配置文件可以为每一个出站代理设置ip-version选项，文档中描述为"设置节点使用 IP 版本，可选：dual/ipv4/ipv6/ipv4-prefer/ipv6-prefer,默认使用dual"。
想了解一下这个选项是设置客户端连接**代理服务器节点**时使用的IP版本，还是代理服务器连接**最终访问网站**所使用的IP版本？

**Comment by nb5p** [comment, marked as answer]
后者

**Reply by leic4u** [reply]
> 后者

反了，是前者。
这里控制的是 mihomo 如何与节点建立连接。

至于节点如何访问网站，得看服务端节点的出站配置是怎么配的。

**Comment by jelly21fish** [comment]
有人说，这影响的是内核连接节点服务器时使用的ip，而不是节点访问谷歌奈飞时的ip，不知道哪个说法正确。如果是前者，想改变后者的话应该怎么做呢？希望开发者能解答一下~
